### DISCUSSION TITLE
Is it possible to use specific nitro hooks once the preset already registered the same hook?

### DISCUSSION THREAD

**Discussion by igorjacauna** [nitrojs/nitro#1376] [Q&A]
I would like to use the `compiled` hook but if I use a preset, my hook overwrites the preset hook. So, the preset hook is not called, only my hook configured in `nitro.options.hooks.compiled`

**Comment by Hebilicious** [comment]
Can you open an issue with a reproduction please?

**Reply by igorjacauna** [reply]
Created #1382

**Comment by igorjacauna** [comment]
Like mentioned in issue #1382 , it is a limitation with Nitro config merging.
